src/drivers/intel/ssp/ssp.c

```diff
@@ -659,6 +659,74 @@ static int ssp_set_config(struct dai *dai,
 	return ret;
 }
 
+/* SSP configuration update before start */
```

Comment:
This needs to be quite verbose explaining why we need to do this and things to take care about.